.github/workflows/python-publish.yml

```diff
@@ -5,8 +5,28 @@ on:
     types: [published]
     
 jobs:
+  pytest:
+    name: Run pytest
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10.13"
```

Comment:
I think we should try to support python 3.9 which is a sensible min requirement, 3.9 will recieve support until 2025-10 (https://www.python.org/downloads/)